hack/k8s/example-ro.template.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: datamon-ro-demo
+spec:
+  selector:
+    matchLabels:
+      app: datamon-ro-demo
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: datamon-ro-demo
+    spec:
+      containers:
+      - name: demo-shell
+        image: fuse-demo-shell:latest
+        imagePullPolicy: "Never"
+        command: ["/bin/tini"]
+        args: ["--", "/bin/bash"]
+        stdin: true
+        tty: true
+        volumeMounts:
+        - mountPath: /tmp/mount
+          name: fuse-mountpoint
+          mountPropagation: "HostToContainer"
+      - name: datamon-sidecar
+        image: fuse-demo-sidecar:latest
+        imagePullPolicy: "Never"
+        command: ["datamon"]
+        args: ["bundle", "mount", "--bundle", "1Kbtk5Mo0M8ck2lcnuhMNYiV9DO", "--repo", "ransom-datamon-test-repo", "--destination", "/tmp", "--mount", "/tmp/mount"]
+        securityContext:
+          privileged: true
```

Comment:
What is the minimum securityContext? Does SYS_ADMIN work?

Reply:
i've tried both

```
        securityContext:
          allowPrivilegeEscalation: true
          capabilities:
            add:
              - SYS_ADMIN
```
and
```
        securityContext:
          capabilities:
            add:
              - SYS_ADMIN
```

and neither allows access to fuse

Reply:
ok